rust/valuation.rs

```diff
@@ -0,0 +1,28 @@
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct ValuationRequestProto {
```

Comment:
Did you have to hand code these @samiam376 ?

Reply:
no these are auto generated